cartesian_impedance_msgs/msg/SetCartesianImpedance.msg

```diff
@@ -1,10 +1,13 @@
 # Here we combine all the parameters message definitions into a single message
 # that needs to be send to the controller in order to adjust the Cartesian Impedance Parameters
 
-string               tool
-CartesianStiffness   stiffness
-CartesianDamping     damping
-MaxCartesianVelocity max_cart_vel
-MaxControlForce      max_ctrl_force
-MaxPathDeviation     max_path_deviation
-NullSpace            null_space_params
+string                 tool
+CartesianStiffness     stiffness
+#use this if the damping is in Ns/m, Ns/rad
```

Comment:
Yes, I meant dampening.

From a usability perspective, any publisher and any subscriber should be able to talk and the semantic intent of the message should be clear.  In other words, the messages sent should always mean the same thing regardless of who is producing/consuming it.  

I think you should pick convention and then make sure subscribers can translate that convention into whatever is expected by the controller.  For example, if you pick Ns/m, then the drive node for the iiwa will require a critical dampening parameter(s) in order to translate into a unit-less dampening.  While this may seem like it makes the driver more complex, it does so for the sake of usability and consistent semantics.